README.md

```diff
@@ -1,121 +1,68 @@
 
 ![DroneQuest Tests](https://api.travis-ci.org/DroneQuest/drone-quest.svg "Tests are passing! ... right?")
 
-# Drone Quest
 
-Our goal is to use terminal commands to send commands to a Parrot AR Drone 2.0 and successfuly make the connect.
-The end goal of this project is to hopefully be able to fly a drone using a Leap Motion Controller.
+# DroneQuest
 
+### What is DroneQuest?
+DroneQuest is a collaborative project between students of Codefellows JavaScript 401 and Python 401 classes.
+The Python portion of this project allows users to connect any controller they want to an API endpoint and send commands to an
+[AR Parrot Drone](http://www.parrot.com/usa/products/ardrone-2/).
 
-## Core Features:
-- Create a python-based API that will communicate with a front-end server
-- Send terminal commands to the drone, that the drone will then execute
-- Send useful drone information to a front-end user interface
-- Create a module that anyone with a drone can pip install, then use to control their personal drone
-	
-## Stretch Goals:
-- Connect a Leap Motion Controller to control the drone
-- Send commands via the front-end web application we have created
-- Use VR goggles that will connect to the drone cameras 
+### How do I use DroneQuest?
 
+Dependencies:
+- [Python 2.7](https://www.python.org/download/releases/2.7/)
+- [Bottlepy](http://bottlepy.org/docs/dev/index.html)
+- [numpy](http://www.numpy.org/)
 
-## Installation
+Unix Systems:
+```
+you@dronequest:~ $ git clone https://github.com/DroneQuest/drone-quest.git
+you@dronequest:~ $ cd drone-quest
+you@dronequest:~ $ python setup.py install
+you@dronequest:~ $ ./runapp.sh
+```
+
+Windows Systems:
+```
+C:\Users\you> git clone https://github.com/DroneQuest/drone-quest.git
+C:\Users\you> cd drone-quest
+C:\Users\you> python setup.py install
+C:\Users\you> runapp.bat
 ```
```

Comment:
We should make a "Mac" section since not everyone knows that Mac OS is unix-based